src/com/trollworks/gcs/advantage/Advantage.java

```diff
@@ -48,7 +48,7 @@
 public class Advantage extends ListRow implements HasSourceReference, Switchable {
     @Localize("Advantage")
     @Localize(locale = "de", value = "Vorteil")
-    @Localize(locale = "ru", value = "Преимущество")
+    @Localize(locale = "ru", value = "ÐŸÑ€ÐµÐ¸Ð¼ÑƒÑ‰ÐµÑ�Ñ‚Ð²Ð¾")
```

Comment:
This looks like your editor mangled the UTF-8 encoding

Reply:
Agreed.   I think I changed the encoding AFTER I had downloaded the code.   Sorry.   How can I revert this?

Reply:
As long as your editor/IDE is set to the right mode, you should be able to just copy and paste the deleted line above and then re-push it.

Reply:
I believe I fixed this with the latest commit.